scipy/special/tests/test_support_alternative_backends.py

```diff
@@ -89,8 +101,5 @@ def test_support_alternative_backends(xp, data, f_name_n_args):
     # To compensate, we also check that the root-mean-square error is
     # less than eps**0.5.
     ref = xp.asarray(ref, dtype=dtype_xp)
-    xp_assert_close(res, ref, rtol=eps**0.2, atol=eps*20,
+    xp_assert_close(res, ref, rtol=eps**0.5, atol=eps*20,
```

Comment:
We can now tighten the relative tolerance since we're not using such extreme arguments.